backend/backend.go

```diff
@@ -539,9 +545,11 @@ func (backend *Backend) Coin(code coinpkg.Code) (coinpkg.Coin, error) {
 			nil)
 	case erc20Token != nil:
 		etherScan := etherscan.NewEtherScan("1", backend.etherScanHTTPClient)
-		coin = eth.NewCoin(etherScan, erc20Token.code, erc20Token.name, erc20Token.unit, "ETH", params.MainnetChainConfig,
+		blockBook := blockbook.NewBlockbook("1", backend.blockbookHTTPClient)
```

Comment:
Blockbook is able to deliver the results for the address for all tokens too in one shot, along with the main ETH address. I think it would be worth it to reduce the load on the backend. `tokenBalances` could be used as the `detail` param only if there are active ERC20 tokens too.  

Do you have a plan in mind for how to achieve that? If you use a separate blockBook instance here this will not be possible. but maybe you have something in mind.

Reply:
I haven't thought about yet in details, but my idea would be to change the interface (or have the wrapper not implement `rpcclient.interface`) so that the wrapper exposes one `Balance` that takes a parameter to mean if we either want only ETH, only ERC20, or both (assuming all three cases make sense, if only the latter does, there is no need for a parameter) and tries to call the Blockbook instance first (with a single call in any case), and if it fails back to Etherscan, it just makes separate calls as needed